CHANGELOG.md

```diff
@@ -4,9 +4,14 @@ This project attempts to adhere to [Semantic Versioning](http://semver.org).
 
 ## [Unreleased]
 
+### Changed
+
+- only boost similarity in Jaro-Winkler once the Jaro similarity exceeds 0.7
```

Comment:
Would it be worth mentioning this in the README and/or the function documentation? Since you said that you've also seen implementations that always boost.

Reply:
Yes that probably makes sense.